.chloggen/instrumentationOptions.yaml

```
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: instrumentation
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add an option that lets the user define a custom instrumentation spec. The user-defined spec will be merged into the default spec.
+# One or more tracking issues related to the change
+issues: [1873]
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

```suggestion
subtext: |
- The configuration structure for operator auto-instrumentation has changed. All settings must now be defined under `instrumentation.spec`.
- Any configuration directly under `instrumentation.*` (outside of `spec`) will cause an error during helm operations.
- If you use `.Values.operator.enabled=true` and have overrides for `instrumentation` settings, please review the [upgrade guidelines](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/UPGRADING.md#01280-to-01290)
```
haven't verified the link

---

thanks, yeah, verified, link is functional